### PR TITLE
fix: simplifies exceptions management

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/TR-1265

### CODE REVIEW

File: src/qtism/common/ContentPackageExceptionInterface.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * @license GPLv2
+ */
+declare(strict_types=1);
+
+namespace qtism\common;
+
+/**
+ * The ContentPackageExceptionInterface intended to help unify the exception handlers behavior
+ */
+interface ContentPackageExceptionInterface
```
Comment (wazelin) [inline src/qtism/common/ContentPackageExceptionInterface.php:29]:
```suggestion
interface PackageContentExceptionInterface
```

**Review by wazelin — CHANGES_REQUESTED** [review]

File: src/qtism/common/ContentPackageExceptionInterface.php (hunk 1 above)
Comment (boajer) [reply src/qtism/common/ContentPackageExceptionInterface.php:29]: @wazelin Should https://www.php-fig.org/bylaws/psr-naming-conventions/ be respected?

Comment (quintanilhar) [reply src/qtism/common/ContentPackageExceptionInterface.php:29]: What do you think about using the lib name in the base exception class just like [Guzzle does](https://github.com/guzzle/guzzle/blob/master/src/Exception/GuzzleException.php)?

Something like: `QtiSdkException`

**Review by quintanilhar — COMMENTED** [review]
I think a good standard is to use the lib name for the base exception to make it easy to track it down.

**Review by quintanilhar — APPROVED** [review]
LGTM 👍🏻

**Review by julien-sebire — APPROVED** [review]
All good for me. Nice job!

**Pull request merged**